bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_crb-systemauthenticated-scc-restricted-v2.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
@stlaz please add the same annotations that you have on the ClusterRole to this role binding

Reply:
I added the annotations from there, please see if those are the correct ones

Reply:
Thank you, that lgtm